config/default/jenkins-infra.yaml

```diff
@@ -149,6 +149,24 @@ jenkins:
                           }
                         }
                       }
+                  - script: >
+                      multibranchPipelineJob('docker-packer') {
+                        displayName "Packer Docker Build"
+                        branchSources {
+                          github {
+                            id('2020101901')
+                            scanCredentialsId('github-access-token')
+                            repoOwner('jenkins-infra')
+                            repository('docker-packer')
+                            includes('main')
```

Comment:
Wouldn't it be better to build every branches but with different behavior like only publishing the image from main